c7n/mu.py

```diff
@@ -1129,7 +1130,7 @@ def add(self, func):
         log.debug('Creating cwe rule target for %s on func:%s' % (
             self, func_arn))
 
-        self.client.put_targets(
+        RuleRetry(self.client.put_targets,
```

Comment:
retry on not found doesn't make sense here

Reply:
Got it, thanks.